lmctl/drivers/lm/security.py

```diff
@@ -71,12 +74,25 @@ def __init__(self, auth_address, username=None, password=None, client_id=None, c
             client_builder.zen_api_key_auth(username=self.__username, api_key=self.__api_key, zen_auth_address=self.__auth_address)
         elif self.__auth_mode.lower() == TOKEN_AUTH_MODE:
             client_builder.token_auth(token=self.__token)
+        elif self.__auth_mode == OKTA_MODE:
+            if self.__username is not None:
+                # Using password auth
+                if self.__client_id is not None:
+                    client_builder.okta_user_pass_auth(username=self.__username, password=self.__password,
```

Comment:
If `self.__username is not None` but `self.__client_id is None` then nothing happens - should this be considered an error scenario?

Reply:
Addressed.

Reply:
I think the error needs to be clearer: `TNCO environment cannot be configured without a client_id when using auth_mode={OKTA_MODE}`

Reply:
Addressed